src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+  const keys = Object.keys(robot);
+  const res = {};
```

Comment:
Checklist:
3.[NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)